htdocs/api/v0.0.1/Login.php

```diff
@@ -75,9 +75,13 @@ function handlePOST()
         $login = $this->getLoginAuthenticator();
 
         if ($login->passwordAuthenticate($user, $password, false)) {
-            $this->JSON = array(
-                           "token" => $this->getEncodedToken($user),
-                          );
+            $token = $this->getEncodedToken($user);
+            if (!empty($token)) {
+                $this->JSON = array("token" => $token);
+            } else {
+                $this->header("HTTP/1.1 500 Internal Server Error");
```

Comment:
This HTTP response header is being sent twice, once here and once from getEncodedToken. IIRC that will generate an apache error.